doxia-doc-renderer/pom.xml

```diff
@@ -172,6 +172,7 @@ under the License.
                   <excludes>
                     <exclude>org.apache.maven.doxia:doxia-module-markdown</exclude>
                     <exclude>org.nibor.autolink:autolink</exclude>
+                    <exclude>com.vladsch.flexmark:*</exclude>
```

Comment:
Is this still necessary if we have bumped to Java 7?

Reply:
It's not necessary - just removed it.